src/resources/js/tribe-common.js

```diff
@@ -135,6 +136,27 @@ tribe_auto_sysinfo.ajax = {
 
 	};
 
+	my.init_tooltip = function () {
+
+		$( document ).on( 'click', '.tribe-tooltip', function () {
+
+			var $tooltip = $( this ).closest( '.tribe-tooltip' );
+			var add = true;
+			if ( $( $tooltip ).hasClass( 'active' ) ) {
```

Comment:
`$tooltip` is already a jQuery var - no need to wrap it again?

You could also shorten this to:
`var add = $tooltip.hasClass( 'active' );`
But this is fine too 😄